src/DssSpell.sol

```diff
@@ -40,56 +43,34 @@ contract DssSpellAction is DssAction {
-        // https://vote.makerdao.com/polling/QmUMyywc#poll-detail
-        // Note: only on mainnet
-
-        // ---- MOMC Parameter Changes ----
-        // https://vote.makerdao.com/polling/QmUMyywc#poll-detail
-
-        // Stability Fee Changes
-
-        //Increase WSTETH-B Stability Fee to 0.5%
-        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_FIVE_PCT_RATE, true);
+    uint256 constant ZERO_SEVENTY_FIVE_PCT_RATE = 1000000000236936036262880196;
+    uint256 constant ONE_PCT_RATE               = 1000000000315522921573372069;
+    uint256 constant ONE_FIVE_PCT_RATE          = 1000000000472114805215157978;
 
-        // Reduce RETH-A Stability Fee to 0.5%
-        DssExecLib.setIlkStabilityFee("RETH-A", ZERO_FIVE_PCT_RATE, true);
+    uint256 constant MILLION = 10 ** 6;
+    uint256 constant RAY     = 10 ** 27;
 
-        // Reduce WBTC-A Stability Fee to 1.75%
-        DssExecLib.setIlkStabilityFee("WBTC-A", ONE_SEVENTY_FIVE_PCT_RATE, true);
+    address immutable MCD_SPOT = DssExecLib.spotter();
 
-        // Reduce WBTC-B Stability Fee to 3.25%
-        DssExecLib.setIlkStabilityFee("WBTC-B", THREE_TWENTY_FIVE_PCT_RATE, true);
-
-        // line changes
-
-        // Increase CRVV1ETHSTETH-A line to 100 million DAI
-        // Note: only on mainnet
-
-        // Increase RETH-A line to 10 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("RETH-A", 10 * MILLION);
-
-        // Increase MATIC-A line to 15 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("MATIC-A", 15 * MILLION);
-
-        // Increase DIRECT-COMPV2-DAI line to 30 million DAI
-        // Note: only on mainnet
-
-        // ---- SF-001 Contributor Vesting ----
-        // Note: only on mainnet
-
-        // Bump changelog
-        DssExecLib.setChangelogVersion("1.14.9");
+    function actions() public override {
+        // Stablecoin vault offboarding
+        // https://vote.makerdao.com/polling/QmemXoCi#poll-detail
+        Fileable(MCD_SPOT).file("USDC-A",   "mat", 15 * RAY); // 1500% collateralization ratio
+        Fileable(MCD_SPOT).file("PAXUSD-A", "mat", 15 * RAY);
+        Fileable(MCD_SPOT).file("GUSD-A",   "mat", 15 * RAY);
+        DssExecLib.updateCollateralPrice("USDC-A");
+        DssExecLib.updateCollateralPrice("PAXUSD-A");
+        DssExecLib.updateCollateralPrice("GUSD-A");
+
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmXGgakY#poll-detail
+        DssExecLib.setIlkStabilityFee("ETH-C", ZERO_SEVENTY_FIVE_PCT_RATE, true);
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_SEVENTY_FIVE_PCT_RATE, true);
+        DssExecLib.setIlkStabilityFee("WBTC-C", ONE_PCT_RATE, true);
+        DssExecLib.setIlkStabilityFee("YFI-A", ONE_FIVE_PCT_RATE, true);
+        DssExecLib.setIlkAutoLineDebtCeiling("RETH-A", 20 * MILLION);
+        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 4 * MILLION);
+        // DssExecLib.setIlkAutoLineDebtCeiling("DIRECT-COMPV2-DAI", 70 * MILLION); ilk does not exist in Goerli
```

Comment:
Should there also be a commented-out line for the AAVE D3M debt ceiling raise? From the linked poll:

* Increase the **Aave v2 D3M** Maximum Debt Ceiling by 15 million DAI from 5 million DAI to **20 million DAI**.

Reply:
Ah I see now the discussion in the new-spells channel, never mind.